### ISSUE TITLE
No way to specify `cargo clippy` options inherited from `cargo check`

### ISSUE THREAD

**Issue by str4d** [auguwu/clippy-action#117]
The `cargo clippy` command is built up thusly:

https://github.com/auguwu/clippy-action/blob/7e80a5cd50fdf05a4885add273e8ba8590119f80/src/clippy.ts#L56-L80

This allows constructing commands of the form:
```
$ cargo clippy --message-format json [--all-features] -- [inputs.args...] [-Ffoo...] [-Dfoo...] [-Wfoo...] [-Afoo...]
```

However, the full format allowed is `cargo clippy [options] [--] [<opts>...]`. In particular, `inputs.args` enables arbitrary extension of `<opts>`, but there is no way to configure `options`. This means that users of `clippy-action` can't e.g. configure `--tests` (to run Clippy on test code) or `--all-targets` (to run Clippy on tests, benches, etc.) or any other `cargo check` option that is compatible.

**Comment by auguwu** [comment]
I can add another action argument for it, if you can, create a PR for it?

**Comment by auguwu** [comment]
this is now included in v1.3.0!
